<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TunerMainWindow</class>
 <widget class="QMainWindow" name="TunerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pupae Thermal Device Tuner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QWidget" name="parameters_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="controller_parameters_groupbox">
         <property name="font">
          <font>
           <bold>false</bold>
          </font>
         </property>
         <property name="title">
          <string>Controller Parameters</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="ControllerParametersWidget" name="controller_parameters_widget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="system_values_groupbox">
         <property name="title">
          <string>System Values</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="SystemValuesWidget" name="system_values_widget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="plot_groupbox">
         <property name="title">
          <string>Plot</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="PlotSettingsWidget" name="plot_settings_widget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="plot_area_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>15</horstretch>
        <verstretch>15</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_file_save_data"/>
    <addaction name="action_file_save_figure"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="action_device_connection"/>
    <addaction name="action_device_information"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_device_connection">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="action_file_save_data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="action_device_information">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="action_file_save_figure">
   <property name="text">
    <string>Save Figure</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ControllerParametersWidget</class>
   <extends>QWidget</extends>
   <header>.controller_parameters_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SystemValuesWidget</class>
   <extends>QWidget</extends>
   <header>.system_values_widget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotSettingsWidget</class>
   <extends>QWidget</extends>
   <header>.plot_settings_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
